### ISSUE TITLE
Type annotatins for 'columns'

### ISSUE THREAD

**Issue by MomIsBestFriend** [pandas-dev/pandas#30449]
ref https://github.com/pandas-dev/pandas/pull/30398#discussion_r361198265

**Comment by jbrockmendel** [comment]
incontinence -> inconsistent?

**Comment by MomIsBestFriend** [comment]
> incontinence -> inconsistent?

Yes! ```incontinence``` sounded wrong, but I couldn't figure out why, but now I know why.

**Comment by jbrockmendel** [comment]
ok good, otherwise that would have been an exceptionally high-priority issue to address
